### DISCUSSION TITLE
Curious about the future of FreedomGPT fnt tokens

### DISCUSSION THREAD

**Discussion by kbuwel** [ohmplatform/FreedomGPT#143] [Q&A]
I am getting fnt tokens with 5700x3d.
I'm converting only as much credit as I need.
The remaining tokens are being transferred to my Ethereum wallet.
The white paper states that tokens transferred to an Ethereum wallet cannot be converted to credits.
I convert less into credits.
I would like to know how the tokens sent to my Ethereum wallet are used.
What can you do with fnt other than converting it to coins like Bitcoin or receiving it as fiat 
currency?
And on what exchanges will fnt tokens be available for trading?
There are numerous exchanges around the world, and some prohibit trading to or from certain 
countries.
For example, Korean coin exchanges do not allow coins to be transferred to coin exchanges in other 
countries other than the US coin exchange.
How will fnt token support these exchanges?
My question is about how fnt tokens will be used in the future, not now.

**Comment by Darcy-P** [comment]
I believe you're incorrect in that FNT in a wallet can be used to purchase credits, but it takes some time to process the transaction, whereas FNT earned off-chain in your FreedomGPT account can be simply converted to credits instantly. 
To do this you simply go to 'Buy More' (Credits), and select 'Pay with Freedom Network Token (FNT)' as your payment method, and then follow the instructions to send the FNT payment from your wallet.
The information in the draft whitepaper (currently dated February 20th, 2024) may be a little outdated as much has been developed in the FreedomGPT app since then.

In my observation their goal included establishing FNT as a tradable currency, and decentralizing access to AI; in turn, enabling a method to access to AI for those who can not simply buy credits with fiat (by acquiring FNT and using it to buy credits), or may not be able to access centralized AI due to network policy.
They are both a work in progress.
